pkg/model/api/v1/datasource.go

```diff
@@ -36,7 +36,7 @@ func FilterDatasource[T DatasourceInterface](kind string, defaultDTS *bool, list
 
 type DatasourceInterface interface {
 	GetMetadata() modelAPI.Metadata
-	GetDTSSpec() DatasourceSpec
+	GetDatasourceSpec() DatasourceSpec
```

Comment:
or GetSpec ? :smile:

Reply:
ah indeed, better :D

Reply:
Oh actually not possible as-is, theres's already a GetSpec